File: Java/MDC.java
public class MDC {
    public static void main(String[] args) {
        int a = 36;
        int b = 48;

        int mdc = calcularMDC(a, b);

        System.out.println("O MDC de " + a + " e " + b + " = " + mdc);
    }

    public static int calcularMDC(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }
}

File: Javascript/MDC.js
function calcularMDC(a, b) {
  while (b !== 0) {
    const temp = b;
    b = a % b;
    a = temp;
  }
  return a;
}

const numero1 = 36;
const numero2 = 48;

const mdc = calcularMDC(numero1, numero2);

console.log(`O MDC de ${numero1} e ${numero2} é: ${mdc}`);
